site/content/docs/user/ingress.md

```diff
@@ -26,6 +26,7 @@ by the ingress controller `nodeSelector`.
 1. [Create a cluster](#create-cluster)
 2. Deploy an Ingress controller, the following ingress controllers are known to work:
     - [Contour](#contour)
+    - [Ambassador](#ambassador)
```

Comment:
nit: maybe move it above contour so it's in alphabetical order

Reply:
Sure. I'll fix that.